examples/shaders/sky-shader/src/lib.rs

```diff
@@ -3,18 +3,16 @@
 #![cfg_attr(
     target_arch = "spirv",
     no_std,
-    feature(register_attr, lang_items),
-    register_attr(spirv)
+    feature(register_tool, lang_items),
+    register_tool(rust_gpu)
```

Comment:
```suggestion
```
(the tool module provided by `spirv-builder` and `lang_items` hasn't been needed in a while)

Also, we should make sure we run `cargo +stable check -p sky-shader -p mouse-shader` *at least* in CI (well, it passes before this PR, thanks to `cfg_attr`, but still).